### PR TITLE
Us21 sign up page css

### PR DESCRIPTION
**Pull request by AnnesophieMarchet**

### CODE REVIEW

File: client/src/components/Form/Form.jsx [hunk 1]
```diff
@@ -23,88 +23,134 @@ function Form() {
...
 
   return (
-    <div className="container-form">
-      <form onSubmit={handleSubmit}>
-        <div>
-          <h3>Je suis:</h3>
-          <button type="button">Candidat</button>
-          <button type="button">Une Entreprise</button>
+    <div className=" flex flex-col w-5/6 mx-auto max-w-sm ">
+      <form onSubmit={handleSubmit} className="">
```
Comment (bastien-fournier) [inline client/src/components/Form/Form.jsx:27]: class name empty

**Review by bastien-fournier — CHANGES_REQUESTED** [review]

File: client/src/components/Form/Form.jsx (hunk 1 above)
Comment (AnnesophieMarchet) [reply client/src/components/Form/Form.jsx]: className fix

**Review by wilderLPM — APPROVED** [review]
Great job

File: client/src/pages/SignUpPage.jsx [hunk 2]
```diff
@@ -1,10 +1,14 @@
+import BurgerMenu from "../components/BurgerMenu";
 import Form from "../components/Form/Form";
 import LogoExternatic from "../components/LogoExternatic";
 
 export default function SignUpPage() {
   return (
     <>
-      <LogoExternatic />
+      <BurgerMenu />
+      <div style={{ width: "120px", height: "90px" }} className="flex mx-auto">
```
Comment (StoicFrenchDev) [inline client/src/pages/SignUpPage.jsx:9]: Style should be moved to the CSS file.

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]

**Review by StoicFrenchDev — APPROVED** [review]

**Review by bastien-fournier — APPROVED** [review]
All good

**Pull request merged**